accepted/2021/Globalization/GlobalizationSize.md

```diff
@@ -0,0 +1,235 @@
+
+GlobalizationNative_LastIndexOf
+- same as GlobalizationNative_IndexOf w/ NSBackwardsSearch
+
+GlobalizationNative_StartsWith
+- can be implemented trough GlobalizationNative_CompareString?
+
+GlobalizationNative_EndsWith
+- can be implemented trough GlobalizationNative_CompareString?
+
+GlobalizationNative_GetSortKey
+- wcsxfrm_l?
+
+GlobalizationNative_ChangeCase
+- NSString uppercaseStringWithLocale
+- NSString lowercaseStringWithLocale
+
+GlobalizationNative_ChangeCaseInvariant
+- NSString uppercaseString
+- NSString lowercaseString
+
+GlobalizationNative_ChangeCaseTurkish
+- Implemented in S.G.N using u_tolower/u_toupper, so easy to replicate
+
+GlobalizationNative_InitOrdinalCasingPage
+-  Implemented in S.G.N using u_toupper, so easy to replicate
+
+GlobalizationNative_GetCalendars
+GlobalizationNative_GetCalendarInfo
+GlobalizationNative_EnumCalendarInfo
+GlobalizationNative_GetLatestJapaneseEra
+GlobalizationNative_GetJapaneseEraStartDate
+- NSCalendar (TBD: details)
+```
+
+### ICU4X
+
+It is interesting enough, there is a newly launched open-source project called [ICU4X](https://github.com/unicode-org/icu4x#icu4x----) for introducing globalization support libraries (similar to ICU). ICU4X has the exact goals we need to achieve with our concerned scenarios.
+
+```
+ICU4X will provide an ECMA-402-compatible API surface in the target client-side platforms, including the web platform, iOS, Android, WearOS, WatchOS, Flutter, and Fuchsia, supported in programming languages including Rust, JavaScript, Objective-C, Java, Dart, and C++.
+
+The design goals of ICU4X are:
+- Small and modular code
+- Pluggable locale data
+- Availability and ease of use in multiple programming languages
+- Written by i18n experts to encourage best practices
+```
+
+That is exactly what we need (or what we are trying to achieve).
+Eric Erhardt already did some investigation and arranged a meeting with the project committee to learn more about the project and the status of the project. It was very helpful getting in touch and learning more about this project. It is a promising project managed by experts and using the CLDR data which means it still sticking with the standards.
+
+The catch here is ICU4X still in the early stages and still not providing all functionality needed by .NET for globalization support. The project committee is willing to get and prioritize requests from different parties. We already communicated in the meeting what functionality .NET currently using from ICU4C and want to see that supported in ICU4X. For example, collation support is one of the topics we brought to the committee's attention.
+
+### Plan Suggestion
+
+ICU4X is the promising path we should pursue to address all concerned scenarios. We can wait a little bit more to get the needed missing functionality implemented in the project and integrate it to .NET as we did with ICU4C.
```

Comment:
If we plan to be a consumer of ICU4x, should we also plan on being on investor? Where are the resources for ICU4X coming from?

Reply:
ICU4X current effort is driven by engineers from Google and Mozilla under the umbrella of a [Unicode Charter](https://github.com/unicode-org/icu4x/blob/main/docs/process/charter.md).

We're curating the project for easy onboarding and put effort to stay inclusive for [new contributors](https://github.com/unicode-org/icu4x/blob/main/CONTRIBUTING.md).
You can find the [PRD](https://github.com/unicode-org/icu4x/blob/main/docs/process/prd.md) and [1.0 Roadmap](https://github.com/unicode-org/icu4x/blob/main/docs/process/roadmap.md) and if you'd like to contribute anything else, we'd be excited to work with you and incorporate it into our planning.